frontends/pytorch/e2e_testing/torchscript/main.py

```diff
@@ -0,0 +1,40 @@
+#  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See https://llvm.org/LICENSE.txt for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import argparse
+
+from torch_mlir.torchscript.e2e_test.framework import run_tests, report_results
+from torch_mlir.torchscript.e2e_test.registry import GLOBAL_TEST_REGISTRY
+
+# Available test configs.
+from torch_mlir.torchscript.e2e_test.configs import (
+    RefBackendTestConfig, NativeTorchTestConfig, TorchScriptTestConfig
+)
+
+# Import tests to register them in the global registry.
+import basic
```

Comment:
You've stumped me: what is this doing?

Reply:
Oh, I see: this is a module in the current package. See if you can re-phrase this as a relative import (`from .. import basic`) for clarity.

Reply:
I started with that and spent a few hours debugging why it didn't work. The problem is that relative imports don't work if you do `python main.py`. The best I could come up with was to cd into the parent directory (`frontend/pytorch/e2e_testing`) and do `python -m torchscript.main`, which is fairly uncomfortable since it can only be run from one directory (compare with running `python frontends/pytorch/e2e_testing/torchscript/main.py` from the root level).

Any thoughts on how to best structure a "main binary" in Python would be very welcome.

For now, adding a comment to clarify this, and will adjust based on best practices:
```
# TODO: Use a relative import.
# That requires invoking this file as a "package" though, which makes it
# not possible to just do `python main.py`. Instead, it requires something
# like `python -m tochscript.main` which is annoying because it can only
# be run from a specific directory.
```